### ISSUE TITLE
Use SViper without short reads

### ISSUE THREAD

**Issue by tungufoss** [smehringer/SViper#3]
I want to be able to run sviper without giving it a short read bam file, to get the consensus :)

**Comment by smehringer** [comment]
Hi :)
So you want to give it a long read BAM file and an VCF file and have sviper output a multi FASTA file containing all the consensus sequences for the appropriate positions?

**Comment by tungufoss** [comment]
Yes, exactly. I'm working with an tandem repeat insertion that we are worried would get polished away if we use the short reads as well.

**Comment by smehringer** [comment]
@tungufoss Alright :)

Last question: Do you want a flanking reference region around the consensus (analogous to the polished consensus I use)?
The consensus sequence will be around 800 bp long, if you plan to remap the consensus to the reference you might want the flanking region because it will create an artificial long read (18.000bp) and will ensure correct remapping to the corresponding reference position.

**Comment by hannespetur** [comment]
Why don't we have both? It could be an option in the project's graceful argument parser.

**Comment by smehringer** [comment]
Please checkout the current develop branch to test the new feature and tell me if it works.

Happy Holidays :)
